Makefile

```diff
@@ -4,6 +4,13 @@ all: lkmpg.tex
 	bibtex main >/dev/null || echo
 	pdflatex -shell-escape $< 2>/dev/null >/dev/null
 
+html: lkmpg.tex
+	make4ht --shell-escape --utf8 --format html5 --config html.cfg --output-dir html lkmpg.tex
```

Comment:
Can we specify the path of the generated HTML as `html/index.html`? Thus, we can deploy the directory later.

Reply:
I currently don't find a way to assign the output filename when using make4ht
Maybe add
`mv ./html/lkmpg.html ./html/index.html into` Makefiles.
Also can I add
`rm -f *.xref *.svg *.tmp *.html *.css *.4ct *.4tc *.dvi *.lg *.idv`
at the end of html target in makefile so only the html directory retained?

Reply:
Use symbolic link instead of `mv`.